lib/config/rules/functionality.js

```diff
@@ -0,0 +1,80 @@
+// Functionality Specific:
+// These rules catch common errors in JS programming or
+// otherwise confusing constructs that are prone to producing bugs:
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Bans the use of specific functions or global methods.
+  ban: false,
```

Comment:
Can you use quotes for all keys, for consistency? The back and forthing makes my brain hurt :P You can turn off the rule that requires unquoted keys like we do in the `eslint-plugin-shopify` repo.